### PR TITLE
cmd/bosun: Collect ack status info by notification

### PR DESCRIPTION
**Pull request by kylebrandt**

### CODE REVIEW

File: cmd/bosun/sched/check.go [hunk 1]
```diff
@@ -306,6 +310,8 @@ func (s *Schedule) CollectStates() {
 	severityCounts := make(map[string]map[string]int64)
 	abnormalCounts := make(map[string]map[string]int64)
 	ackStatusCounts := make(map[string]map[bool]int64)
+	ackByNotificationCounts := make(map[string]map[bool]int64)
```
Comment (captncraig) [inline cmd/bosun/sched/check.go]: the map[bool] here is still odd. I think I would rather see ackCounts and unackedCounts maps[string]int

File: cmd/bosun/sched/check.go [hunk 2]
```diff
@@ -323,10 +329,35 @@ func (s *Schedule) CollectStates() {
 		ackStatusCounts[alert.Name][true] = 0
 		activeStatusCounts[alert.Name][true] = 0
 	}
+	for notificationName, _ := range s.Conf.Notifications {
+		unAckOldestByNotification[notificationName] = time.Unix(1<<63-62135596801, 999999999)
```
Comment (captncraig) [inline cmd/bosun/sched/check.go]: wtf is this number?

File: cmd/bosun/sched/check.go (hunk 1 above)
Comment (kylebrandt) [inline cmd/bosun/sched/check.go]: I believe it follows the existing pattern though, no?

File: cmd/bosun/sched/check.go (hunk 2 above)
Comment (kylebrandt) [inline cmd/bosun/sched/check.go]: Mac value of time.Time, so I can loop and get the min. http://stackoverflow.com/a/32620397/107156

**Pull request merged**